xblocks_contrib/word_cloud/word_cloud.py

```diff
@@ -96,9 +115,196 @@ def workbench_scenarios():
             ),
         ]
 
-    @staticmethod
-    def get_dummy():
+    def student_view(self, context=None):  # pylint: disable=W0613
+        """
+        Create primary view of the WordCloudXBlock, shown to students when viewing courses.
+        """
+        frag = Fragment()
+        frag.add_content(resource_loader.render_django_template(
+            "templates/word_cloud.html", {
+                'display_name': self.display_name,
+                'instructions': self.instructions,
+                'element_class': self.scope_ids.block_type,
+                'element_id': uuid.uuid1(0),
```

Comment:
@farhan didn't we decide to use block_id here instead uuid.uid1(0)?

Reply:
We decided to use any as far as its working and unique identifier.
We have used `uuid.uid1(0)` because:
1. DoneXBlock is using it as an identifier
2. We were avoiding it as we were facing issues in test cases but we have managed a work around in test cases
3. `block_id` was problematic in one case which as far as I remember while doing the development

So we are good to go with `uuid` I think 
cc: @ttqureshi